internal/controllers/addon/phase_ensure_catalogsource_networkpolicy_test.go

```diff
@@ -256,98 +255,31 @@ func TestEnsureCatalogSourceNetworkPolicy_Adoption(t *testing.T) {
 	for name, tc := range map[string]struct {
 		ActualNetworkPolicy *networkingv1.NetworkPolicy
```

Comment:
if we remove adoptionstrategy, what are we testing in this specific test?

Reply:
It is more of a smoke/regression test. It is there just to roughly check and make sure the function is not broken by providing different inputs with different owners.